builds.yml

```diff
@@ -192,7 +192,7 @@ env:
   - SKIP_BACKGROUND_INITIALIZATION: false
 
   # TODO(ritave): Move ManifestV3 into a feature?
```

Comment:
safe to remove the todo here? I doubt we'll ever move it to a "feature".

Reply:
Good catch ! removed